airflow/utils/log/base_airflow_logging.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class BaseAirflowLogging(object):
+    """
+    A base class for setting up custom airflow logging behaviors.
```

Comment:
Same comment as above, i.e. why are we doing this differently from the way airflow_local_settings works for the policy file. Also AIRFLOW_HOME should be PYTHONPATH (in general the comment should be similar to the one in settings.py for "def policy")